tests/report_test.py

```diff
@@ -38,7 +38,7 @@ def _generate_report(input_file):
 def reports():
     """ Fixture that analyses all reports"""
     pool = Pool(cpu_count())
-    input_files = sorted([f for f in TESTDATA_INPUTS.iterdir()])
+    input_files = sorted([f for f in TESTDATA_INPUTS.iterdir() if f.name != 'environments.sol.o'])
```

Comment:
Why does it need to be disabled here?

Reply:
The jsonification?

Reply:
Yup.

Reply:
Yup

PS: I made this comment like 3 times... github is dying on me... :(

Reply:
Okay, and its the debug section that causes this right?
Because if the jsonification fails/takes too long on the report.as_json() for any of the test contracts then we can't just ignore this

Reply:
Yup.

It also significantly increases runtime of the tests without this calldata improvement, so it's really an issue that should be resolved somehow. Perhaps open a new issue?

Reply:
I created one #614